tests/system/action/user/test_update_temporary.py

```diff
@@ -85,77 +75,78 @@ def test_update_all_fields(self) -> None:
         assert model.get("vote_delegations_$_from_ids") == ["222"]
         assert model.get("vote_delegations_from_ids") is None
 
-    def test_update_vote_weight(self) -> None:
-        self.create_model(
-            "user/111",
-            {"username": "username_srtgb123", "meeting_id": 222},
-        )
-        response = self.client.post(
-            "/",
-            json=[
-                {
-                    "action": "user.update_temporary",
-                    "data": [{"id": 111, "vote_weight": "1.500000"}],
-                }
-            ],
-        )
-        self.assert_status_code(response, 200)
-        model = self.get_model("user/111")
-        assert model.get("vote_weight") == "1.500000"
+    # TODO: Re-enabled these tests after vote weight is reimplemented
```

Comment:
Please stop commenting out tests with TODO's. They tend to stick around and noone cares about it. So, what has to be done here? If possible, please fix it in this PR since it belongs here since this apparently broke the tests.

Reply:
I understand the problem but in the current case it should be an extra task. It is not just a small fix but an extension of fields that were not there before (i. e. non relational template fields for vote_weight). I can open an issue for this and, if you like, remove the tests completely. But please let us solve this thing in an extra step.

Reply:
See https://github.com/OpenSlides/openslides-backend/issues/422